test/spec/spec.form.js

```diff
@@ -353,6 +355,13 @@ describe('Form Controller', function () {
             req.form.values.bool.should.equal(true);
         });
 
+        it('creates a validate array when validate is a string or field options exist', function () {
+            form.post(req, res, cb);
+            expect(form.options.fields.bool.validate).to.be.undefined;
+            form.options.fields.place.validate.should.eql(['required']);
+            form.options.fields.options.validate.length.should.equal(1);
+        });
+
```

Comment:
Merge it! ;-)

On 17 June 2016 09:27:35 BST, Gavin Boulton <notifications@github.com> wrote:
>> @@ -353,6 +355,13 @@ describe('Form Controller', function () {
>>              req.form.values.bool.should.equal(true);
>>          });
>>  
>> +        it('creates a validate array when validate is a string or
>field options exist', function () {
>> +            form.post(req, res, cb);
>> +           
>expect(form.options.fields.bool.validate).to.be.undefined;
>> +           
>form.options.fields.place.validate.should.eql(['required']);
>> +           
>form.options.fields.options.validate.length.should.equal(1);
>> +        });
>> +
>
>I know what you mean but it also feels a bit overkill.
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/UKHomeOffice/passports-form-controller/pull/48/files/932fce8ecd9aef52dcd07ae2a6a4f21757306a54#r67475615